<html><h1>Validation FAILED.</h1><p>Build Time: 25 minutes 26 seconds 29 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/e13af5f6a4000541fbc22e9959ce7216b5021e30">e13af5f6a4000541fbc22e9959ce7216b5021e30</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [echo] java.lang.Exception: Found 1 formatting issues:
     [echo] 1: ./commerce-product-test/src/testIntegration/java/com/liferay/commerce/product/service/test/CPDefinitionLocalServiceTest.java expected:&lt;...e;
     [echo] import org.junit.[]Rule;
     [echo] import org.jun...&gt; but was:&lt;...e;
     [echo] import org.junit.[Ignore;
     [echo] import org.junit.]Rule;
     [echo] import org.jun...&gt;
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:439)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:267)
     [echo] 
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * What went wrong:
     [echo] Execution failed for task ':checkSourceFormatting'.
     [echo] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [echo] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [echo] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [echo] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/81//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>48 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/81//testReport">here</a>.</h5></div></html>

--- a/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java
+++ b/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java
@@ -35,6 +35,7 @@ import com.liferay.portal.kernel.test.util.CompanyTestUtil;
 import com.liferay.portal.kernel.test.util.RandomTestUtil;
 import com.liferay.portal.kernel.test.util.ServiceContextTestUtil;
 import com.liferay.portal.kernel.test.util.UserTestUtil;
+import com.liferay.portal.kernel.util.ListUtil;
 import com.liferay.portal.kernel.workflow.WorkflowConstants;
 import com.liferay.portal.test.rule.Inject;
 import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
@@ -468,6 +469,8 @@ public class CommerceAccountLocalServiceTest {
 		Assert.assertEquals(
 			userCommerceAccounts2.toString(), 2, userCommerceAccounts2.size());
 
+		userCommerceAccounts2 = ListUtil.sort(userCommerceAccounts2);
+
 		CommerceAccount userCommerceAccount2a = userCommerceAccounts2.get(0);
 		CommerceAccount userCommerceAccount2b = userCommerceAccounts2.get(1);
 
@@ -487,6 +490,8 @@ public class CommerceAccountLocalServiceTest {
 
 		Assert.assertEquals(
 			userCommerceAccounts3.toString(), 3, userCommerceAccounts3.size());
+
+		userCommerceAccounts3 = ListUtil.sort(userCommerceAccounts3);
 
 		CommerceAccount userCommerceAccount3a = userCommerceAccounts3.get(0);
 		CommerceAccount userCommerceAccount3b = userCommerceAccounts3.get(1);
@@ -527,6 +532,8 @@ public class CommerceAccountLocalServiceTest {
 
 		Assert.assertEquals(
 			userCommerceAccounts5.toString(), 4, userCommerceAccounts5.size());
+
+		userCommerceAccounts5 = ListUtil.sort(userCommerceAccounts5);
 
 		CommerceAccount userCommerceAccount5a = userCommerceAccounts5.get(0);
 		CommerceAccount userCommerceAccount5b = userCommerceAccounts5.get(1);

--- a/commerce-openapi-admin-test/src/testIntegration/java/com/liferay/commerce/openapi/admin/resource/test/PortalContextProvider.java
+++ b/commerce-openapi-admin-test/src/testIntegration/java/com/liferay/commerce/openapi/admin/resource/test/PortalContextProvider.java
@@ -76,22 +76,22 @@ public abstract class PortalContextProvider {
 	protected static final String USER = "test@liferay.com";
 
 	@Inject
-	protected static CompanyLocalService companyLocalService;
+	protected CompanyLocalService companyLocalService;
 
 	@Inject
-	protected static EROrganizationLocalService erOrganizationLocalService;
+	protected EROrganizationLocalService erOrganizationLocalService;
 
 	@Inject
-	protected static GroupLocalService groupLocalService;
+	protected GroupLocalService groupLocalService;
 
 	@Inject
-	protected static OrganizationLocalService organizationLocalService;
+	protected OrganizationLocalService organizationLocalService;
 
 	@Inject
-	protected static RoleLocalService roleLocalService;
+	protected RoleLocalService roleLocalService;
 
 	@Inject
-	protected static UserLocalService userLocalService;
+	protected UserLocalService userLocalService;
 
 	private static final String _ROOT_END_POINT_SUFFIX = "/o/commerce-admin/";
 

--- a/commerce-product-test/src/testIntegration/java/com/liferay/commerce/product/service/test/CPDefinitionLocalServiceTest.java
+++ b/commerce-product-test/src/testIntegration/java/com/liferay/commerce/product/service/test/CPDefinitionLocalServiceTest.java
@@ -40,7 +40,6 @@ import org.frutilla.FrutillaRule;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -117,7 +116,6 @@ public class CPDefinitionLocalServiceTest {
 			WorkflowConstants.STATUS_INACTIVE, cpInstance.getStatus());
 	}
 
-	@Ignore
 	@Test
 	public void testAddCPDefinitionWithDefaultInstanceAndNoSKUs()
 		throws Exception {
@@ -179,7 +177,6 @@ public class CPDefinitionLocalServiceTest {
 			WorkflowConstants.STATUS_INACTIVE, cpInstance.getStatus());
 	}
 
-	@Ignore
 	@Test
 	public void testAddCPDefinitionWithDefaultInstanceAndSKUs()
 		throws Exception {
@@ -354,17 +351,17 @@ public class CPDefinitionLocalServiceTest {
 	public final FrutillaRule frutillaRule = new FrutillaRule();
 
 	@Inject
-	private static CPDefinitionLocalService _cpDefinitionLocalService;
+	private CPDefinitionLocalService _cpDefinitionLocalService;
 
 	@Inject
-	private static CPDefinitionOptionRelLocalService
+	private CPDefinitionOptionRelLocalService
 		_cpDefinitionOptionRelLocalService;
 
 	@Inject
-	private static CPInstanceLocalService _cpInstanceLocalService;
+	private CPInstanceLocalService _cpInstanceLocalService;
 
 	@Inject
-	private static CPOptionLocalService _cpOptionLocalService;
+	private CPOptionLocalService _cpOptionLocalService;
 
 	@DeleteAfterTestRun
 	private Group _group;

--- a/commerce-product-type-virtual-web/src/main/resources/META-INF/resources/definition_virtual_setting/terms_of_use.jsp
+++ b/commerce-product-type-virtual-web/src/main/resources/META-INF/resources/definition_virtual_setting/terms_of_use.jsp
@@ -22,7 +22,6 @@ CPDefinitionVirtualSettingDisplayContext cpDefinitionVirtualSettingDisplayContex
 CPDefinitionVirtualSetting cpDefinitionVirtualSetting = cpDefinitionVirtualSettingDisplayContext.getCPDefinitionVirtualSetting();
 
 JournalArticle journalArticle = cpDefinitionVirtualSettingDisplayContext.getJournalArticle();
-boolean termsOfUseRequired = BeanParamUtil.getBoolean(cpDefinitionVirtualSetting, request, "termsOfUseRequired");
 
 long termsOfUseJournalArticleResourcePrimKey = BeanParamUtil.getLong(cpDefinitionVirtualSetting, request, "termsOfUseJournalArticleResourcePrimKey");
 

--- a/commerce-product-type-virtual-web/src/main/resources/META-INF/resources/terms_of_use.jspf
+++ b/commerce-product-type-virtual-web/src/main/resources/META-INF/resources/terms_of_use.jspf
@@ -26,6 +26,11 @@
 <liferay-ui:error exception="<%= CPDefinitionVirtualSettingTermsOfUseException.class %>" message="please-select-an-existing-web-content-or-enter-terms-of-use-content" />
 
 <aui:fieldset>
+
+	<%
+	boolean termsOfUseRequired = BeanParamUtil.getBoolean(cpDefinitionVirtualSetting, request, "termsOfUseRequired");
+	%>
+
 	<aui:input label="enable-terms-of-use" name="termsOfUseRequired" type="toggle-switch" value="<%= termsOfUseRequired %>" />
 </aui:fieldset>
 

